marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/CreateAndUpdateModelTest.java

```diff
@@ -230,6 +339,23 @@ private void assertIndexDeployment() {
         });
     }
 
+    private void getIndexes(Boolean isIndexDeleted) {
+        String json = new DatabaseManager(getHubClient().getManageClient()).getPropertiesAsJson(getHubConfig().getDbName(DatabaseKind.FINAL));
+        Database db = new DefaultResourceMapper(new API(getHubClient().getManageClient())).readResource(json, Database.class);
+        List<ElementIndex> rangeIndexes = db.getRangeElementIndex();
+        if(!isIndexDeleted) {
+            assertEquals("someProperty", rangeIndexes.get(rangeIndexes.size() - 2).getLocalname());
```

Comment:
Assertions that assume that the index is somewhere in a list are brittle - they could break. We don't care what position in the list the index is in, we just care whether it exists or not. 

To achieve that, get the Database object, then add a private method like this:

```
private boolean rangeIndexExists(Database db, String name, String type) 
```

And then just iterate through each index object in the Database, and if one with the given name/type are found, return true. Else return false.